chef_master/source/delivery_manage_secrets.rst

```diff
@@ -64,4 +80,44 @@ From there, begin using the secrets by calling the ``get_project_secrets`` metho
      end
   end
 
-which is part of the ``publish.rb`` recipe in the ``delivery-truck`` cookbook: https://github.com/chef-cookbooks/delivery-truck/blob/master/recipes/publish.rb#L91-L103.
+This example is part of the ``publish.rb`` recipe in the ``delivery-truck`` cookbook: https://github.com/chef-cookbooks/delivery-truck/blob/master/recipes/publish.rb#L91-L103.
+
+Using a Chef Vault
+=====================================================
+
+This section describes how to use Chef vault in Chef Automate.
+
+Create a Chef Vault
+=====================================================
+In order to use Chef vaults you must follow hierarchical naming standard for your Chef vaults under the `workflow-vaults` data bag:
+
+.. code-block:: javascript
+
+   <ENT>
+   <ENT>-<ORG>
+   <ENT>-<ORG>-<PROJECT>
+
+For example, if the ``chef`` enterprise has a ``cookbooks`` organization with a ``mysql`` project, it would have the naming schema:
+
+.. code-block:: javascript
+
+   chef
+   chef-cookbook
+   chef-cookbook-mysql
+
+During the creation of a Chef vault, the data in these vaults are merged into a single Ruby hash. Any duplicate key names will be merged as follows:
+   - ``<ENT>-<ORG>-<PROJECT>`` will overwrite ``<ENT>-<ORG>`` and ``<ENT>``.
+   - ``<ENT>-<ORG>`` will overwrite ``<ENT>``.
+
+Using an Encrypted Data Bag
```

Comment:
This is not an encrypted data bag. This is a Chef Vault... 🤔 We have to fix this.